.aspell.en.pws

```diff
@@ -59,3 +59,4 @@ CPUs
 MiB
 RBAC
 YAML
+loki
```

Comment:
Please use the capitalised version, `Loki`. We should not be mixing and matching so want to catch these. It's the name of a tool like Prometheus. I imagine it may even be allowed as the name of a popular Norse god.